packages/browser/src/modules/logto-client.test.ts

```diff
@@ -0,0 +1,19 @@
+import { createRequester } from '@logto/js';
+
+import LogtoClient from './logto-client';
+
+describe('LogtoClient', () => {
+  test('constructor', () => {
+    expect(
+      () =>
+        new LogtoClient({
+          endpoint: 'https://logto.dev/oidc/.well-known/openid-configuration',
+          clientId: 'client_id_value',
+          scopes: ['openid', 'offline_access'],
```

Comment:
`openid` and `offline_access` should be reserved scopes thus no need to pass when create a client. actually we can hide the param for now

Reply:
Updated. PTAL @gao-sun